### ISSUE TITLE
JN.1+S:F456L with C6968T branch (67 seqs)

### ISSUE THREAD

**Issue by DailyCovidCases** [cov-lineages/pango-designation#2682]
From branch 79 of https://github.com/sars-cov-2-variants/lineage-proposals/issues/1253
JN.1+C6968T+S:F456L
Query: C6968T, T22928C, C18894T
No. of seqs: 30 (Canada 1 China 3 France 1 Singapore 3 South Korea 2 Spain 2 Sweden 1 UK 10 USA 7)
First: EPI_ISL_19051595, 2024-3-23
Latest: PP934275.1, 2024-6-10
[usher](https://nextstrain.org/fetch/genome.ucsc.edu/trash/ct/singleSubtreeAuspice_genome_19086_423570.json?f_userOrOld=uploaded%20sample)
![Screenshot (131)](https://github.com/cov-lineages/pango-designation/assets/92421581/abb9cb57-7654-4ca2-84c5-fc5689a2c83b)

**Comment by FedeGueli** [comment]
thanks for proposing but i suggest to close this down , the query catches multiple lineages
<img width="910" alt="Screenshot 2024-07-02 alle 18 50 49" src="https://github.com/cov-lineages/pango-designation/assets/87669813/817424ec-fe5b-4615-84e5-f426b5d43ba1">
https://nextstrain.org/fetch/genome-test.gi.ucsc.edu/trash/ct/singleSubtreeAuspice_genome_test_462b3_42fab0.json?f_userOrOld=uploaded%20sample&label=id:node_6939135
the one proposed here seems very small:
<img width="1133" alt="Screenshot 2024-07-02 alle 18 51 19" src="https://github.com/cov-lineages/pango-designation/assets/87669813/e4c66517-b46d-4b74-ac92-08d631e1fda4">

When a multiple lineages issue is open it is like a notebook where initial queries are annotated, sometimes they are very good , while more usually they then have to be updated to exclude convergent lineages, so when you go to propose a lineage and you see that the sequences are widespread in the tree better stopping and highlighiting that in the comments before proposing it. thank you for your work anyway

**Comment by aviczhl2** [comment]
> When a multiple lineages issue is open it is like a notebook where initial queries are annotated, sometimes they are very good , while more usually they then have to be updated to exclude convergent lineages, so when you go to propose a lineage and you see that the sequences are widespread in the tree better stopping and highlighiting that in the comments before proposing it. thank you for your work anyway

Yeah. But it is quite shocking that C6968T is so convergent as a synonym mutation.

**Comment by FedeGueli** [comment]
> > When a multiple lineages issue is open it is like a notebook where initial queries are annotated, sometimes they are very good , while more usually they then have to be updated to exclude convergent lineages, so when you go to propose a lineage and you see that the sequences are widespread in the tree better stopping and highlighiting that in the comments before proposing it. thank you for your work anyway
> 
> Yeah. But it is quite shocking that C6968T is so convergent as a synonym mutation.

agree

**Comment by DailyCovidCases** [comment]
Tracked in [tracker79](https://cov-spectrum.org/explore/World/AllSamples/Y2024/variants?nucMutations=C6968T%2CT22928C%2CC18894T&nextcladePangoLineage=JN.1*&)

**Comment by FedeGueli** [comment]
> C6968T, T22928C, C18894T

did you update the query? @DailyCovidCases if not it is not monophyletic as highlighted in a comment above. please check
